restapi/src/main/java/io/stargate/web/docsapi/exception/ErrorCode.java

```diff
@@ -63,6 +60,13 @@
           "The parameter `page-size` is limited to %d.",
           DocsApiConfiguration.DEFAULT.getMaxPageSize())),
 
+  DOCS_API_GENERAL_UPGRADE_INVALID(
+      Response.Status.BAD_REQUEST, "The collection cannot be upgraded in given manner."),
```

Comment:
Does the user actually have a choice for how the upgrade is done? I'd just state `The collection cannot be upgraded`. WDYT?

Reply:
nit: why `GENERAL` in the enum name?